flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/TableStoreFactory.java

```diff
@@ -65,8 +69,8 @@
 
     @Override
     public void onCreateTable(Context context, boolean ignoreIfExists) {
-        Map<String, String> options = context.getCatalogTable().getOptions();
```

Comment:
It is OK to keep options.
All options have been enriched except `enrichOptions`, we don't need to emphasize it.

Reply:
From the developer's perspective, we all know that `#onCreate` and `#onDrop` get called only after `#enrichOptions` is executed. While from the API layer, it's hard to tell they have an order (except taking a reference to Flink's codebase). That's why I change the name.

Reply:
`onCreate` and other methods has its duties, it doesn't even need to have the concept of enriched, which I think is redundant.